exasol/ds/sandbox/lib/pretty_print.py

```diff
@@ -0,0 +1,23 @@
+from datetime import datetime, timedelta
+
+
+def elapsed(start: datetime, round_to_seconds=True) -> str:
```

Comment:
Their is package Stopwatch which might help with that. We use it already in other packages.

Reply:
You only resolved the comment without fixing or answering. Please check again.